migration/mule/cache-scope-with-salesforce-contacts/README.md

```diff
@@ -0,0 +1,56 @@
+# Cache Scope With Salesforce Contacts 
+
+This example illustrates the basic concept of data caching using Salesforce.
+
+### Assumptions ###
+
+This document assumes that you are familiar with WSO2 EI and the 
+[Integration Studio interface](https://ei.docs.wso2.com/en/latest/micro-integrator/overview/quick-start-guide/). To 
+increase your familiarity with Integration Studio, consider completing one or more 
+[WSO2 EI Tutorials](https://ei.docs.wso2.com/en/latest/micro-integrator/use-cases/integration-use-cases/).
+
+### Example Use Case
+
+The very basic concept of caching consists of temporary storing a result of some operations, in our use case it is caching fetched data from Salesforce. The example illustrates caching when we ask for a data once again, the result is returned directly from cache instead of making a new call to Salesforce.
+
+### Set Up and Run the Example
+
+1. Start WSO2 Integration Studio ([Installing WSO2 Integration Studio](https://ei.docs.wso2.com/en/latest/micro-integrator/develop/installing-WSO2-Integration-Studio/)).
+2. In your menu in Studio, click the **File** menu. In the File menu select the **Import...** item.
+3. In the Import window select the **Existing WSO2 Projects into workspace** under **WSO2** folder.
+4. Browse and select the file path to the downloaded sample of this Github project 
+(`integration-studio-examples/migration/mule/cache-scope-with-salesforce-contacts/CacheScopeWithSalesforceContactsRegistry`) anf click **finish**.
+5. Let's add the Salesforce REST connector into the workspace. Right click on the **CacheScopeWithSalesforceContacts** and select 
+**Add or Remove Connector**. Keep the **Add connector** option selected and click **Next>**. Search for 'salesforce' using the 
+search bar and click the download button located at the bottom right corner of the Salesforce REST connector. Click **Finish**.
+6. Follow these [steps](https://ei.docs.wso2.com/en/latest/micro-integrator/references/connectors/salesforce-rest-connector/sf-access-token-generation/) to generate the Access Tokens for Salesforce and obtain the Access Token, and Refresh Token.
+7. Open the **ContactsAPI.xml** under 
+**cache-scope-with-salesforce-contacts/CacheScopeWithSalesforceContacts/src/main/synapse-config/api/** directory. 
+Configure the following properties with the previously obtained values.
+    - Access Token
+    - Refresh Token
+    - API URL (e.g.: https://<INSTANCE>.salesforce.com)
```

Comment:
This does not shown correctly in enriched view. I got an error when i tried it just following the instructions. 

![image](https://user-images.githubusercontent.com/4967046/80800921-699f7780-8bc8-11ea-9045-e4560c784c40.png)

Reply:
Fixed